test/attributes.test.ts

```diff
@@ -49,16 +56,106 @@ describe("Attributes", () => {
         const getAttributesResponse = await client1.attributes.getValidAttributes({});
         expect(getAttributesResponse).toBeSuccessful(ValidationSchema.ConnectorAttributes);
     });
+
+    test("should create a Repository Attribute", async () => {
+        const attribute = await client1.attributes.createRepositoryAttribute({
+            content: {
+                value: {
+                    "@type": "GivenName",
+                    value: "AGivenName"
+                }
+            }
+        });
+        expect(attribute.isSuccess).toBe(true);
+    });
```

Comment:
this is copy paste from the first test (`should create a repository attribute` lowercase)